lxc/storage.go

```diff
@@ -88,6 +88,10 @@ lxc storage delete [<remote>:]<pool>
 lxc storage edit [<remote>:]<pool>
     Edit storage pool, either by launching external editor or reading STDIN.
 
+lxc storage <pool-name> btrfs.mount_options "<mount option>,<mount option>,<mount option>"
+    add btrfs mount options to btrfs storage pool
+    EXAMPLE: lxc storage <pool-name> btrfs.mount_options "rw,nospace_cache,lazytime,user_subvol_rm_allowed,autodefrag"
+
```

Comment:
Cool, closing for now then.

For the "make i18n && make update-po" you need to be at the root of the lxd code tree (where the Makefile file is). If that step is causing you too much trouble, just fix the rest of the comments and I'll do that part for you.